### PR TITLE
add --login for shell to support directly run from xcode

### PR DESCRIPTION
**Pull request by bestwnh**
now WebIDLToSwift can directly run and debug from xcode

### CODE REVIEW

File: Sources/WebIDLToSwift/Shell.swift [hunk 1]
```diff
@@ -26,7 +26,7 @@ enum Shell {
         // print("*** running script: \(script)")
         let task = Process()
         task.standardError = FileHandle.standardError
-        task.arguments = ["-c", script]
+        task.arguments = ["--login", "-c", script]
```
Comment (kateinoigakukun) [inline Sources/WebIDLToSwift/Shell.swift:29]: I don't think we really need shell here just to run npm scripts.

Comment (bestwnh) [reply Sources/WebIDLToSwift/Shell.swift:29]: If without this, the Xcode report can't find `swiftformat`. Same with the npm start.

Comment (kateinoigakukun) [reply Sources/WebIDLToSwift/Shell.swift:29]: The right way to do it is tell PATH env var to Xcode scheme

Comment (bestwnh) [reply Sources/WebIDLToSwift/Shell.swift:29]: I am not familiar with environment variables, can you provide the values that need to be set?
And what is the down side of adding `--login`?

Comment (kateinoigakukun) [reply Sources/WebIDLToSwift/Shell.swift:29]: It's not yours fault but I think using shell here at all is not the right way. For example, what happens when your shell is not zsh but bash.

Comment (bestwnh) [reply Sources/WebIDLToSwift/Shell.swift:29]: Could you provide some key class or method names to provide a better solution? I may try to make a PR to improve it.

**Comment by kateinoigakukun** [comment]
Closing in favor of https://github.com/swiftwasm/WebAPIKit/commit/beec492c7b9c36647e98455cfabfa4ba9bd2b8cf

You can configure PATH in Xcode by 
Screenshot 2024-05-08 at 16 44 48

**Pull request closed without merging**